runtime/autoload/provider/clipboard.vim

```diff
@@ -55,6 +54,14 @@ function! s:cmd_ok(cmd) abort
   return v:shell_error == 0
 endfunction
 
+function! s:split_cmd(cmd) abort
+  if type(a:cmd) == v:t_string
+    return split(a:cmd, " ")
+  else
+    return a:cmd
+  endif
```

Comment:
ternary is much shorter without losing clarity.

    return (type(a:cmd) == v:t_string) ? split(a:cmd, " ") : a:cmd